docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,283 @@
+
+The `TwoOrgsChannel` provides name of the `SampleConsortium` that is used to create the channel. As a result, the ordering service defined in the `TwoOrgsOrdererGenesis` profile becomes channel consenter set. The `Application` section creates the part of the channel that governs the channel's peer organizations. Both organizations from the consortium, Org1 and Org2, are included as channel members. The channel uses `V2_0` as the application capabilities, and uses the default policies to govern how peer organizations will interact with the channel. The application channel also uses the default policies from the **Channel** section and enables `V2_0` as the channel capability level.
+
+## Channel policies
+
+Because channels are used as a private method of communication between organizations, any changes that are made to the channel need to be agreed by other channel members. A channel would not be useful if an organization could join the channel and read the data on the ledger without getting the approval of other organizations. Because part of the channel ledger is stored on the ordering nodes that create new blocks, channel members also need to select the ordering nodes that participate in the channel. Any changes to the **structure** of the channel need to be approved by the set of organizations that is specified by channel policies.
+
+Policies also govern the **processes** of how users interact with the channel, such as the set of organizations that need to approve a chaincode before it can be deployed to a channel, which channel members can invoke a chaincode to write data to the channel, and which actions need to be completed by channel administrators.
+
+Channel policies are important enough that they need to be discussed in their own topic. Unlike other parts of the channel configuration, the policies that govern the channel are determined by by how different sections of the `configtx.yaml` file work together. While channel policies can be configured for any use case with few constraints, this topic will focus on how to use the default policies provided by Hyperledger Fabric. By default, Fabric uses a combination of signature policies, ImplicitMeta policies, and Access Control Lists to determine how channel members participate in the channel processes and update the channel structure. You can learn more about the role of policies in Hyperledger Fabric by visiting the [Policies concept topic](../policies.html).
+
+Each channel member is associated with a set of signature policies. Signature policies read the the digital signatures that are attached to transactions or channel update requests that are submitted to the channel. Each signature policy lists a set of organizations and roles that will satisfy if those signatures are found in the transaction. You can see the set signature policies associated with peer organization Org1 in the **Organizations** section of `configtx.yaml` below:
```

Comment:
wording: Each signature policy lists a set of organizations and roles that will satisfy if those signatures are found in the transaction.

Reply:
How about:  Each signature policy lists a set of organizations and roles whose signatures need to be attached to a transaction in order to satisfy the policy.